commons/src/test/java/org/eclipse/kapua/commons/setting/EnvFriendlyConfigurationTest.java

```diff
@@ -22,7 +22,7 @@
 public class EnvFriendlyConfigurationTest extends Assert {
 
     @Before
-    public void createInstanceOfClasses() {
+    public void createInstanceOfClass() {
```

Comment:
Please move lines with: 
    CompositeConfiguration compositeConfiguration;
    EnvFriendlyConfiguration envFriendlyConfiguration;
to lines 23, 24, so they are before "@Before public void createInstanceofClass()" method.